superbench/common/utils/gen_config.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Utilities for pattern config."""
+from superbench.common.utils import logger
+
+
+def gen_all_nodes_config(n):
```

Comment:
Looks like no extern function will use this function. Maybe we can remove this function.